File: C#/Programming Basics with C#/11. For-Loop - Exercise/06. Divide without remainder/DivideWithoutremainder.cs
using System;

class DivideWithoutremainder
{
    static void Main()
    {
        int n = int.Parse(Console.ReadLine());

        int counterP2 = 0;
        int counterP3 = 0;
        int counterP4 = 0;


        for (int i = 1; i <= n; i++)
        {
            int inputNum = int.Parse(Console.ReadLine());
            if (inputNum % 2 == 0)
            {
                ++counterP2;
            }
            if (inputNum % 3 == 0)
            {
                ++counterP3;
            }
            if (inputNum % 4 == 0)
            {
                ++counterP4;
            }

        }

        Console.WriteLine($"{100.0 * counterP2 / n:F2}%");
        Console.WriteLine($"{100.0 * counterP3 / n:F2}%");
        Console.WriteLine($"{100.0 * counterP4 / n:F2}%");
    }
}

File: C++/Programming Basics with C++/11. Exercise - For-loop/simple_tasks/DivideWithoutRemainder.cpp
#include <iostream>

using namespace std;

int main() {
    int n;
    cin >> n;

    int counterP2 = 0;
    int counterP3 = 0;
    int counterP4 = 0;


    for (int i = 1; i <= n; i++) {
        int inputNum;
        cin >> inputNum;
        if (inputNum % 2 == 0) {
            ++counterP2;
        }
        if (inputNum % 3 == 0) {
            ++counterP3;
        }

        if (inputNum % 4 == 0) {
            ++counterP4;
        }

    }

    cout.setf(ios::fixed);
    cout.precision(2);

    cout << 100.0 * counterP2 / n << '%' << endl;
    cout << 100.0 * counterP3 / n << '%' << endl;
    cout << 100.0 * counterP4 / n << '%' << endl;

    return 0;
}
